app/src/main/java/org/mozilla/focus/screenshot/ScreenshotItemAdapter.java

```diff
@@ -161,19 +161,16 @@ public void onQueryComplete(List result) {
         mIsLoading = false;
     }
 
-    @Override
-    public void onDeleteComplete(int result, long id) {
-        if (result > 0) {
-            if (id < 0) {
-                final int count = mItems.size();
-                mItems.clear();
-                notifyItemRangeRemoved(0, count);
+    public void onItemDelete(long id) {
+        if (id < 0) {
```

Comment:
The id should always be >=0 since we're not feeding -1 like we do in clear_browsing_history. Should be safe to just do the things in else {} here.